### PR TITLE
Caff 18 - Load Start GPS coordinate as Final Waypoint 

### PR DESCRIPTION
**Pull request by tcheung99**
To try this out, run the following with the nav stack (to get the simulated gps readings): 
`roslaunch load_waypoints load_waypoints.launch` 

This PR contains a service node to accomplish requirements listed in #18. If you would like to test it using a simple client node, see the code for `load_waypoints_client.py` here: [https://github.com/UTRA-ART/SLAM-1920/wiki/Information-on-load_waypoints-package](url). 

The client node was omitted from this PR because it isn't yet relevant (but will be used in the future to help cycle through waypoints upon the 'manual trigger' -- see #16).

### CODE REVIEW

File: misc/load_waypoints/srv/WaypointRequest.srv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+int8 waypoint_number
+---
+float32[2] waypoint_coord
+bool valid_request_flag 
+string request_description
+string waypoint_description
```
[the remark this answers is not in the document]
Comment (tcheung99) [reply misc/load_waypoints/srv/WaypointRequest.srv]: Would setting next_waypoint as -1 as invalid/out of range be acceptable?

**Comment by tcheung99** [comment]
Addressed the comments and added back the client node !

**Comment by tcheung99** [comment]
Here's a sample output for what happens when the operation times out: 
image

File: misc/load_waypoints/scripts/load_waypoints_server.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+        # Parse through json data and create list of lists holding all waypoints
+        for waypoint in waypoint_data["waypoints"]:
+            self.all_waypoints[waypoint['id']] = waypoint
+
+        # Call method to wait for transform 
+        self.wait_for_utm_transform()
+
+        # After waiting UTM transform, capture a message from the /gps/fix topic
+        gps_info = rospy.wait_for_message('/caffeine/gps/fix', NavSatFix)
```
[the remark this answers is not in the document]
Comment (tcheung99) [reply misc/load_waypoints/scripts/load_waypoints_server.py]: Update: I tested the change to using namespace in the launch file, but I noticed that my server node doesn't go past the line that waits for gps message

File: misc/load_waypoints/package.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  <maintainer email="trudie.cheung@mail.utoronto.ca">Trudie Cheung</maintainer>
+
+  <license>TODO</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_export_depend>rospy</build_export_depend>
+  <exec_depend>rospy</exec_depend>
+  <exec_depend>message_runtime</exec_depend>
```
Comment (erickmu1) [inline misc/load_waypoints/package.xml:16]: missing
```
tf
std_msgs
sensor_msgs
```
dependencies here

**Pull request merged**